jest.config.js

```diff
@@ -0,0 +1,12 @@
+const base = require("./jest.config.base.js");
+
+module.exports = {
+    ...base,
+    transform: {
+        '\\.jsx?': "<rootDir>/preprocess.js"
+    },
+    projects:

```

Comment:
I dont think that there is any need for anything other than projects key to be defined here. Try deleting `...base` and `transform` keys nad urnning the tests..

Reply:
`transform ` key moved to `jest.config.base.js` file.
Test report is less detailed without `...base` so I kept it in this config.

Reply:
Actually, the extended logs are caused just by `verbose: true` option, but other options seems to be ignored. I would rather have explicitly set the `verbose` key in this file and the `jest.config.base.js` can be used just as a base for other packages. By doing this, you will also be able to set `transform` option globally for all packages (in the base config).

Reply:
`...base` removed from `jest.config.js` in root. Option `verbose: true` added into file.